indexer/services/roundtable/src/tasks/subaccount-username-generator.ts

```diff
@@ -20,7 +20,7 @@ export default async function runTask(): Promise<void> {
   for (const subaccount of subaccountZerosWithoutUsername) {
     for (let i = 0; i < config.ATTEMPT_PER_SUBACCOUNT; i++) {
       const username: string = generateUsernameForSubaccount(
-        subaccount.subaccountId,
+        subaccount.address,
```

Comment:
Even though `subaccountId` is deterministically generated from `address + subaccountNum`, using `address` here is more straightforward and makes it easy to check `generateUsernameForSubaccount` offline.